models/db.py

```diff
@@ -1,9 +1,36 @@
-from gluon.tools import Auth
 db = DAL('sqlite://storage.sqlite',pool_size=1,check_reserved=['all'])
+from gluon.tools import Auth, Service, PluginManager
```

Comment:
Please follow PEP8, "Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants": https://www.python.org/dev/peps/pep-0008/#imports